docs/content/documentation/systems/Materials/phase_field/VanDerWaalsFreeEnergy.md

```diff
@@ -0,0 +1,17 @@
+# VanDerWaalsFreeEnergy
+!syntax description /Materials/VanDerWaalsFreeEnergy
+
+This material defines the Helmholtz free energy density $f$ of a Van der Waals gas.
+
+$$
+F = U - TS = -Nk_BT\left(1+\ln n_Q\left(\frac1n-b\right)\right)-n^2a
```

Comment:
I would put f = F/V on LHS instead of F, then you can put in n = N/V on the RHS so you have everything in terms of n